packages/odyssey-storybook/.storybook/preview.ts

```diff
@@ -99,7 +98,7 @@ const preview: Preview = {
           value: "#f4f4f4",
         },
         {
-          name: "Page Background (dark)",
+          name: "Dark (unused)",
```

Comment:
Don't think we need this currently as there is no current plan in place for dark mode. Would likely just cause confusion IMHO